qa/tasks/vstart_runner.py

```diff
@@ -271,8 +271,10 @@ def put_file(self, src, dst, sudo=False):
         try:
             shutil.copy(src, dst)
         except exception as e:
-            if sys.version_info.major < 3 and e.message.find('are the same '
-               'file') != -1:
+            if sys.version_info.major < 3:
+                if e.message.find('are the same file') != -1:
+                    return
+            else:
```

Comment:
Check exception type  if `sys.version_info.major >= 3`.

Reply:
I am catching `exception`, not `Exception`. `exception` is assigned to shutil.SameFileError just above block of code. See - https://github.com/ceph/ceph/blob/master/qa/tasks/vstart_runner.py#L268.